templates/kv.tmpl

```diff
@@ -19,6 +19,14 @@
     sticky = {{$sticky}}
 {{end}}
 
+{{$healthCheck := Get "" . "/healthcheck/" "path"}}
+{{$interval := Get "30s" . "/healthcheck/" "interval"}}
+{{with $healthCheck}}
```

Comment:
We may move `{{$interval := Get "30s" . "/healthcheck/" "interval"}}` below `{{with $healthCheck}}` ?